src/main/java/com/durov/maks/cinema/model/dto/moviesession/MovieSessionRequestDto.java

```diff
@@ -0,0 +1,33 @@
+package com.durov.maks.cinema.model.dto.moviesession;
+
+import java.time.LocalDateTime;
+
+public class MovieSessionRequestDto {
+    private LocalDateTime date;
```

Comment:
Could you please send a screenshot of the view of your JSON object which is corresponding to this DTO. Just curious.

Reply:
of course
![json](https://user-images.githubusercontent.com/27377208/84373554-6941c580-abe5-11ea-84ab-77c235a519d7.png)